Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/UX - Windows First Run Animation Disabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Disables Windows First Run Animation",
+  "description": "What is it: Disables Windows First Run animation. \n Simeon Notes: Improves the user enrollment experience. \n Learn More: https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-windowslogon#windowslogon-enablefirstlogonanimation",
```

Comment:
**What does this do?**
Disables Windows First Run animation, which displays animation and marketing when a user first signs into Windows. 

**Why should you use this?**
If you want to eliminate marketing materials and non-essential animation during initial sign-in.

**What is the end user impact?**
Users may have an improved experience when opting out of First Run Animation. 

**Learn more**
[Policy CSP - WindowsLogon](If you disable this policy setting, users do not see the animation and Microsoft account users do not see the opt-in prompt for services)